.github/workflows/auto_merge.yml

```diff
@@ -36,9 +36,9 @@ jobs:
           contains(  steps.files.outputs.all, 'data/' )
           || contains(  steps.files.outputs.all, 'config.yml' )
           || contains(  steps.files.outputs.all, 'domain.yml' )
-      run: echo "::set-env name=RUN_TRAINING::true"
+      run: echo "RUN_TRAINING=true" >> $GITHUB_ENV
     - name: Set up Python 3.7
-      if: env.RUN_TRAINING == 'true'
+      if: env.RUN_TRAINING == 'true' 
```

Comment:
Looks like useless space at the end of like :D

Reply:
oh lol yes